.code-samples.meilisearch.yaml

```diff
@@ -175,7 +175,13 @@ update_settings_1: |-
         "synonyms": {
             "wolverine": ["xmen", "logan"],
             "logan": ["wolverine"]
-        }
+        },
+        "typoTolerance": {
+            "minWordSizeForTypos": {
+                "oneTypo": 8,
+                "twoTypos": 10
+        },
```

Comment:
```suggestion
                "twoTypos": 10
                },
        },
```
I think there is a brace missing

Reply:
I just notice that:
```
        "disableOnAttributes": ["title"]
```
Should be on the `typoTolerance` setting

Reply:
You're right. I'll add it after line 185, thanks!